packages/js/rollup.config.cjs.js

```diff
@@ -11,6 +12,9 @@ export default [
       preserveModules: true,
       entryFileNames: '[name].cjs',
     },
-    plugins: [typescript({ outDir: 'dist/cjs', declarationDir: 'dist/cjs/types' })],
+    plugins: [
+      typescript({ outDir: 'dist/cjs', declarationDir: 'dist/cjs/types' }),
+      json()
```

Comment:
why do we need the json plugin?

Reply:
I used it to support JSON import from the previous version, but I am removing it now. thanks